app/src/main/java/org/breezyweather/settings/compose/MainScreenSettingsScreen.kt

```diff
@@ -157,12 +158,26 @@ fun MainScreenSettingsScreen(
                     },
                 )
             }
+
+            val animationsEnabled = !context.isMotionReduced
+                    && (SettingsManager.getInstance(context).backgroundAnimationMode
+                    != BackgroundAnimationMode.DISABLED)
```

Comment:
The second condition will disable cards fade in and other elements which are unrelated to the background animation. Was not part of #136 but makes sense for "gravity sensor" though

Reply:
If the system settings are going to override the app settings, would it make sense to change this into a `switchPreferenceItem`? And then just disable it in the same way if the system animations are disabled? 

It might be more intuitive because that way if animations are enabled system wide the user can either enable or disable them for the app, but if they're disabled system wide the button is also disabled. Having a "follow system" option is somewhat redundant if the user can't enable the option if it's disabled by the system. 

I'll change the other options back to the way they were in the next commit and create a new issue / pull request for them once this is merged.

Reply:
> If the system settings are going to override the app settings, would it make sense to change this into a switchPreferenceItem?

You're talking about the preference for the background animation, right? Currently, the system setting for reduced motion [is only considered](https://github.com/breezy-weather/breezy-weather/blob/a89133c38e3e386f8a189c064fdd41dcd71a002e/app/src/main/java/org/breezyweather/main/fragments/HomeFragment.kt#L112) when the background animation is set to `BackgroundAnimationMode.SYSTEM`. If it is set to `BackgroundAnimationMode.ENABLED`, the system setting is ignored and the background is animated.

By the way, the gravity sensor animation [doesn't adhere](https://github.com/breezy-weather/breezy-weather/blob/a89133c38e3e386f8a189c064fdd41dcd71a002e/app/src/main/java/org/breezyweather/main/fragments/HomeFragment.kt#L169) to the system setting right now. With this change the preference composable would be disabled when reduced motion is active while the gravity sensor animation itelf would still be running by default. _(Some static backgrounds like partly cloudy still have animated elements controlled by the gravity sensor.)_

Reply:
I am. I'll revert the changes to every other option besides the background animation option in the next commit. The linked issue implies that the system settings should take priority over the app settings for background animations. If that's not an inference I should be making please let me know.

Reply:
> The second condition will disable cards fade in and other elements which are unrelated to the background animation. Was not part of #136 but makes sense for "gravity sensor" though

This may be a change for another pull request, but the card fade in also currently tracks the `Context.isMotionReduced` setting in the same way the background animations do. I'm not sure if that's intended or not though. If it is intended should that card behave the same way as the background animations card does? If it's not intended should I create a new issue for it?